### ISSUE TITLE
squad output

### ISSUE THREAD

**Issue by ZhangChuann** [NVIDIA/TensorRT#855]
In TensorFlow run_squad.py, SQuAd output has softmax operation, but in TensorRT squad_output without softmax, Why?

#run_squad.py code in https://github.com/NVIDIA/DeepLearningExamples/blob/master/TensorFlow/LanguageModeling/BERT/run_squad.py
![image](https://user-images.githubusercontent.com/10493733/97307461-b1167000-189a-11eb-91f8-f479c4d01ee3.png)

#TensorRT code  in build.py
def squad_output(prefix, config, init_dict, network, input_tensor):
    """
    Create the squad output
    """

    idims = input_tensor.shape
    assert len(idims) == 5
    B, S, hidden_size, _, _ = idims

    W_out = init_dict[prefix + SQD_W]
    B_out = init_dict[prefix + SQD_B]

    W = network.add_constant((1, hidden_size, 2), W_out)
    dense = network.add_fully_connected(input_tensor, 2, W_out, B_out)

    OUT = network.add_shuffle(dense.get_output(0))
    OUT.second_transpose = (1, 0, 2, 3, 4)
    set_output_name(OUT, prefix, "squad_logits")
    return OUT

**Comment by ttyio** [comment]
Hello @ZhangChuann , thanks for reporting
demobert output logits, and softmax exist when postprocess the logits.
https://github.com/NVIDIA/TensorRT/blob/release/7.2/demo/BERT/helpers/data_processing.py#L470

**Comment by ttyio** [comment]
I will close this, please reopen if you still have question, thanks!
